db/themes/pure

```diff
@@ -1 +1 @@
-https://github.com/rafaelrinaldi/theme-pure
+https://github.com/rafaelrinaldi/pure
```

Comment:
Right. The redirect there was a favor to OMF. Let's hold off on breaking our packages until we support prefixless repo names :)